prac_05/wimbledon.py

```diff
@@ -0,0 +1,42 @@
+"""
+
+"""
+
+
+def main():
+    champions = read_champions()
+    champion_countries = read_champion_countries()
+    display_champions(champions, champion_countries)
+
+
+def read_champions():
+    champions_list = []
```

Comment:
What examples of ours contain `list` in list names?
https://github.com/CP1404/Starter/wiki/Styles-and-Conventions#best-practice-suggestions